docs/DESIGN.md

```diff
@@ -53,19 +53,25 @@ Otherwise, we need to implement each sensible variant of the tool and put it in
 well-named module. This is the case with `Data.Functor`s and `Control.Functor`s
 (recall this [blog post]).
 
-2. If we're adding support for an external resource, we may want a new API. For
-this, we need to think carefully about the protocol of using that external
-resource.  That is, we need to identify a clean set of rules that if we obey,
-we can use the resource to do what we want and not cause system errors. For
-instance, we need to open, use, then close file handles. This protocol is
-obeyed in the API in linear base -- see [examples/Simple/FileIO.hs].
-
-3. The ideas behind new tools that are just now possible with linear types vary
+2. The ideas behind new tools that are just now possible with linear types vary
 and each have unique concepts that are not addressed by a general strategy.
-These should be documented in the section below.
+These should be documented below if one of the following is true:
+
+  * There is an overarching concept that extends beyond a handful of modules
+  * There is an explicit departure away from the direction `base` takes on some
+    tool. (E.g., we decide there should be different laws for some typeclass
+    from `base`.)
 
 ## How Key Pieces are Implemented
 
+### Linear APIs for External Resources
+
+Linear APIs for external resources (in [`System.IO.Resource`]) should obey
+their protocols as much as possible.  For example, the API provided for file
+I/O makes it impossible to use a file handle that has been closed.  It also
+ensures that every file handle is closed. See the example in
+[examples/Simple/FileIO.hs].
```

Comment:
I'm still not sure what this means? Can you just remove this, and will the informational content of the README be different? This sounds all tautological to me.

Reply:
Yes, I'll remove it -- I don't think it's important.